kvmagent/kvmagent/plugins/storage_device.py

```
@@ -306,6 +308,52 @@ def hba_scan(self, req):
+                            if path.startswith('^'):
+                                path = path[1:]
+                            if path.endswith('$'):
+                                path = path[:-1]
+                            if path:
+                                existing_disks.add(path)
+
+        # add OS root disk patterns
+        try:
+            root_disks = ["%s[0-9]*" % d for d in linux.get_physical_disk()]
+            existing_disks = existing_disks.union(root_disks)
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)